src/custom-types.json

```diff
@@ -65,8 +65,9 @@
   "application/javascript": {
     "charset": "UTF-8",
     "compressible": true,
-    "extensions": ["mjs"],
+    "extensions": ["cjs", "mjs"],
     "sources": [
+      "https://nodejs.org/docs/latest-v13.x/api/esm.html#esm_package_scope_and_file_extensions",
```

Comment:
This is required to be a primary source doe these definitions. I'm not sure if it's primary, but more importantly, I don't see it state anywhere that the file extension. cjs is the mime type application/json .

Reply:
thanks for the quick review! and yep, looks like I was completely mistaken. sorry!